### PR TITLE
OSDOCS-2164: Adding release note for Alibaba Cloud IPI

### PR DESCRIPTION
**Pull request by mjpytlak**
CP to 4.10

This PR is the release note for [OSDOCS-2164](https://issues.redhat.com/browse/OSDOCS-2164)

Doc preview

[Installing a cluster on Alibaba Cloud using installer-provisioned infrastructure](https://deploy-preview-40349--osdocs.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-10-release-notes#ocp-4-10-installation-and-upgrade-alibaba-ipi)

### CODE REVIEW

**Comment by mjpytlak** [comment]
@jianli-wei Ahead of a preliminary drop of the 4.10 release notes, requesting QE review. A link to the the installation instructions will be provided as part of GA readiness activities.

**Comment by mjpytlak** [comment]
@jianli-wei As we approach GA, I wanted to follow up on my request to review the release note for Alibaba IPI support. Following up on the request for your review of the release note for Alibaba IPI support. The first draft of the release notes is due 2/18. It would be great if you could take a look by this Friday. Thanks in advance.
CC: @mburke5678

File: release_notes/ocp-4-10-release-notes.adoc [hunk 1]
```diff
@@ -76,6 +76,12 @@ To install a CSI driver on a cluster running on vSphere, you must have the follo
 
 These components are still supported, but are deprecated; support for them will be removed in a future version of {product-title}.
 
+[id="ocp-4-10-installation-and-upgrade-alibaba-ipi"]
+==== Installing a cluster on Alibaba Cloud using installer-provisioned infrastructure
```
Comment (ahardin-rh) [inline release_notes/ocp-4-10-release-notes.adoc:80]: add (Technology Preview) after the heading. An example from 4.9 Release Notes: https://docs.openshift.com/container-platform/4.9/release_notes/ocp-4-9-release-notes.html#ocp-4-9-sandboxed-containers-tp

File: release_notes/ocp-4-10-release-notes.adoc [hunk 2]
```diff
@@ -76,6 +76,12 @@ To install a CSI driver on a cluster running on vSphere, you must have the follo
...
+
+{product-title} {product-version} introduces support for installing a cluster on Alibaba Cloud using installer-provisioned infrastructure. This type of installation lets you use the installation program to deploy a cluster on infrastructure that the installation program provisions and the cluster maintains.
```
Comment (ahardin-rh) [inline release_notes/ocp-4-10-release-notes.adoc:82]: s/introduces support/introduces the ability to install a cluster on Alibaba Cloud using installer-provisioned infrastructure in Technology Preview.

**Comment by mjpytlak** [comment]
@jianli-wei Follow up on my request to review the release note for the Alibaba release note. This is now time sensitive. Docs freeze is on 2/25.

CC @mburke5678 @ahardin-rh

**Comment by jianli-wei** [comment]
> @jianli-wei Follow up on my request to review the release note for the Alibaba release note. This is now time sensitive. Docs freeze is on 2/25.
> 
> CC @mburke5678 @ahardin-rh

LGTM, thanks

**Pull request merged**